command/cmdfactory.py

```diff
@@ -1,3 +1,4 @@
+import os
```

Comment:
If you push to a branch with a pull in progress, it will get updated with the new commits